test/specs/modules/Dropdown/Dropdown.spec.js

```diff
@@ -33,6 +37,61 @@ describe('Dropdown', () => {
     expect(openSpy).to.have.been.calledWith(false);
   });
 
+  it('should close the menu when clicking on option', () => {
+    const wrapper = shallow(DropdownWithRequired, {
+      propsData: {
+        options: [{ text: 'foo', value: 1 }],
+      },
+    });
+    wrapper.vm.setOpen(true);
+    wrapper.find(DropdownItem).trigger('click');
+    wrapper.vm.$nextTick(() => {
+      expect(wrapper.vm.open).to.equal(false);
+    });
+  });
+
+  it('should close the menu when re-clicking on dropdown head', () => {
+    const wrapper = shallow(DropdownWithRequired, {
+      propsData: {
+        placeholder: 'foo',
+        options: [{ text: 'foo', value: 1 }],
+      },
+    });
+    wrapper.vm.setOpen();
+
+    wrapper.find('.text').trigger('click');
+
+    wrapper.vm.$nextTick(() => {
+      expect(wrapper.vm.open).to.equal(false);
+    });
+  });
+
+  it('should not close the menu when clicking on search input', () => {
+    const wrapper = shallow(DropdownWithRequired, {
+      propsData: {
+        search: true,
+        options: [{ text: 'foo', value: 1 }],
+      },
+    });
+
+    wrapper.vm.setOpen(true);
+    wrapper.find('input.search').trigger('click');
+    expect(wrapper.vm.open).to.equal(true);
+  });
+
+  it('should not close the menu when clicking on option when multiple=true', () => {
+    const wrapper = shallow(DropdownWithRequired, {
+      propsData: {
+        multiple: true,
+        options: [{ text: 'foo', value: 1 }],
+      },
+    });
+
+    wrapper.vm.setOpen(true);
+    wrapper.find(DropdownItem).trigger('click');
+    expect(wrapper.vm.open).to.equal(true);
```

Comment:
Same as above, I'd try to see if the `open` class is there or something like that. We might want to change the internal behavior later on and this shouldn't need to change this unit test.